src/clj/vanilla/db/core.clj

```diff
@@ -221,6 +230,8 @@
   ; THIS is the function to setup the initial database!!!!!
   ;
   ;
+<<<<<<< HEAD
```

Comment:
Git merge problem, seems like it tried to auto merger but failed for no reason.

Reply:
Whoops, missed that somehow.  Fixed